src/views/Home/components/DaysLeft/index.jsx

```diff
@@ -6,6 +6,7 @@ import session from 'services/session';
 import { gordonColors } from 'theme';
 import styles from '../Doughnut.module.css';
 import { Card, CardContent, CardHeader, Grid, Typography } from '@mui/material';
+import { NoBackpackSharp } from '@mui/icons-material';
```

Comment:
This looks like an unused import that may have been accidentally added. Please remove it if you don't plan to use it.

Reply:
I'm not sure where it came from-thanks for the catch, I'll remove it